bundle/edu.gemini.spModel.io/src/main/scala/edu/gemini/spModel/io/impl/migration/to2015B/To2015B.scala

```diff
@@ -116,6 +194,36 @@ object To2015B {
     ps
 
   }
+
+  // Proper Motion conversion to degrees/yr
```

Comment:
I kind of lament the loss of units in our `Angle` class but at least with the new `Angle` we know we've got degrees.   I realize PM is angle / year but would it make sense to represent it with an `Angle` instead of a raw `Double`?